projects/components/src/notification/notification.service.test.ts

```diff
@@ -0,0 +1,59 @@
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { NotificationModule, NotificationService } from '@hypertrace/components';
+import { createServiceFactory, mockProvider, SpectatorService } from '@ngneat/spectator';
+import { NotificationComponent, NotificationMode } from './notification.component';
+
+describe('NotificationService', () => {
+  let spectator: SpectatorService<NotificationService>;
+
+  const createService = createServiceFactory({
+    service: NotificationService,
+    imports: [NotificationModule],
```

Comment:
Is the import needed? Shouldn't be to test a service

Reply:
not needed. removed.